test/update.test.js

```diff
@@ -144,7 +144,55 @@ describe('updateAll', function() {
       Product.update({id: id}, newData, function(err, result) {
         testUtil.hasError(err, result).should.not.be.ok();
         testUtil.hasResult(err, result).should.be.ok();
-        var newRev = result[0].rev;
+        var newRev = result[0]._rev;
+        oldRev.should.not.equal(newRev);
+        Product.find(function(err, result) {
+          testUtil.hasError(err, result).should.not.be.ok();
+          testUtil.hasResult(err, result).should.be.ok();
+          newRev.should.equal(result[0]._rev);
+          done();
+        });
+      });
+    });
+  });
+});
+
+describe('updateAttributes', function() {
+  before(function(done) {
+    db = getDataSource();
+
+    Product = db.define('Product', {
+      name: {type: String},
+      description: {type: String},
+      price: {type: Number},
+    }, {forceId: false, updateOnLoad: true});
+
+    db.once('connected', function() {
+      db.automigrate(function(err) {
+        Product.create(bread, done);
+      });
+    });
+  });
+
+  after(cleanUpData);
+
+  it('update an attribute for a model instance', function(done) {
+    var newData = {
+      name: 'bread2',
+      price: 100,
+    };
+    Product.find(function(err, result) {
+      testUtil.hasError(err, result).should.not.be.ok();
+      testUtil.hasResult(err, result).should.be.ok();
+      var id = result[0].id;
+      var oldRev = result[0]._rev;
+      var oldResult = _.cloneDeep(result[0]);
+      oldResult.name = newData.name;
+      var instance = new Product(result[0]);
```

Comment:
maybe `product` is bette than `instance`. 
`var product = new Product(result[0]); `
and `oldResult` is a little misleading, it actually means the new data to be updated.

```js
var updateFields = {
  name: 'bread2',
  price: 100,
}
var newData = _.cloneDeep(result[0]);
updateFields.forEach(updateFields, function(value, key) {
  newData[key] = value;  
});

produce.updateAttributes(updateFields, cb);
```